### PR TITLE
Roadmap Updates

### PR DESCRIPTION
**Pull request by Naros**

### CODE REVIEW

**Comment by Naros** [comment]
Hi @jpechane and @rk3rn3r can both of you take a peak at these changes and if they look solid, we can then merge this sometime this week to update our roadmap accordingly for Q1.

File: roadmap.asciidoc [hunk 1]
```diff
@@ -24,33 +24,26 @@ As our engineering capacity is limited, we're focusing our efforts on a single r
...
-* Debezium UI - Support pass-thru connector configuration parameters
-
 == 2.2 - March 2023
 
 * Support configurable signaling channels
-* Add schema history compaction tool
 * Add new JDBC Sink Connector that natively supports Debezium events
-* Rewrite Cassandra connector to use the common connector framework (run within Debezium Server)
+* New Infinispan Sink connector for Debezium Server
+* New RocketSMQ Sink connector for Debezium Server
```
Comment (jpechane) [inline roadmap.asciidoc]:
```suggestion
* New RocketMQ Sink connector for Debezium Server
```

File: roadmap.asciidoc [hunk 2]
```diff
@@ -24,33 +24,26 @@ As our engineering capacity is limited, we're focusing our efforts on a single r
...
+* Add new SMT to generate diff change events (exposes only changed fields)
+* Rewrite Cassandra connector to use the common connector framework (run within Debezium Server) (tentative)
```
Comment (jpechane) [inline roadmap.asciidoc:34]: This is already done.

**Review by jpechane — COMMENTED** [review]
Commented by me.
@rk3rn3r Could you please review UI stuff?

File: roadmap.asciidoc (hunk 2 above)
Comment (Naros) [reply roadmap.asciidoc:34]: I saw the ISPN sink connector this morning.  I think its worth keeping under 2.2 until that's out the door in March, wouldn't you agree?

Comment (jpechane) [reply roadmap.asciidoc:34]: @Naros  Yes, ISPN is fine. Just Cassandra work is completed.

**Comment by jpechane** [comment]
@Naros Applied, thanks!

**Pull request merged**